biolink-model.yaml

```diff
@@ -202,7 +202,105 @@ enums:
         description: nucleic acid
       "AA":
         description: amino acid
-        
+
+  drug_availability_enum:
+    description: >-
+    permissible_values:
+      "over the counter":
+        description: >-
+          chemical entity is available over the counter without a prescription.
+      "prescription":
+        description:
+          chemical entity is available by prescription.
+
+  FDA_approval_status_enum:
+    description: >-
+    permissible_values:
+      "FDA Approval Phase 1":
+        description: >-
+          Discovery & Development Phase. Discovery involves researchers finding new possibilities
+          for medication through testing molecular compounds, noting unexpected effects from existing treatments,
+          or the creation of new technology that allows novel ways of targeting medical products to sites in the body.
+          Drug development occurs after researchers identify potential compounds for experiments.
+      "FDA Approval Phase 2":
```

Comment:
"FDA Approval Phase 2" is easily confused with "FDA Clinical Research Phase 2", I suggest to call this phase "Preclinical Research Phase".

Reply:
done